make run_docker fails after developer-scripts PR #131

https://github.com/edgexfoundry/developer-scripts/pull/131

The docker-compose files are referenced in `bin/edgex-docker-launch.sh` and reorganizing them has broken `make run_docker` for the moment.  Downloading a compose file and specifying the `$COMPOSE_FILE_PATH` env variable still works.

```
[master] brave:edgex-go jdharms$ make run_docker
bin/edgex-docker-launch.sh
Downloading Docker Compose file...
Starting EdgeX...
ERROR: In file '/tmp/compose-files/docker-compose-edinburgh-1.0.0.yml', the service name 404 must be a quoted string, i.e. '404'.
make: *** [run_docker] Error 1```

--- a/bin/edgex-docker-launch.sh
+++ b/bin/edgex-docker-launch.sh
@@ -20,14 +20,14 @@
 # To override the compose file entirely set the COMPOSE_FILE_PATH environment variable to the full
 # pathname of the compose file you want to use.
 
-GITHUB_PATH=https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master
-VERSION=edinburgh-1.0.0
+GITHUB_PATH=https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/edinburgh/compose-files/
+VERSION=1.0.1
 
 if [ "x$COMPOSE_FILE_PATH" = x ]; then
     if [ $# -ne 0 ] && [ "$1" = "redis" ]; then
-        COMPOSE_FILE=compose-files/docker-compose-redis-$VERSION.yml
+        COMPOSE_FILE=docker-compose-redis-edinburgh-no-secty-$VERSION.yml
     else
-        COMPOSE_FILE=compose-files/docker-compose-$VERSION.yml
+        COMPOSE_FILE=docker-compose-edinburgh-no-secty-$VERSION.yml
     fi
     
     COMPOSE_FILE_PATH=/tmp/$COMPOSE_FILE

